Packages/com.unity.inputsystem/InputSystem/Actions/InputActionParameters.cs

```diff
@@ -0,0 +1,805 @@
+        /// parameter from the object registered as <c>"tap"</c> (which will usually be <see cref="Interactions.TapInteraction"/>).</param>
+        /// <param name="value">New value to assign to the parameter.</param>
+        /// <param name="bindingMask">A binding mask that determines which of <paramref name="actionMap"/>'s <see cref="InputActionMap.bindings"/>
+        /// to apply the override to. By default this is empty which leads to the override to be applied to all bindings in the map.</param>
+        /// <remarks>
+        /// This method both directly applies the new value and also stores the override internally.
+        ///
+        /// You can use this method to set parameters (public fields) on composites, interactions, and processors that are created
+        /// from bindings.
+        ///
+        /// <example>
+        /// <code>
+        /// // Create an action map with two actions.
+        /// var map = new InputActionMap();
+        /// var action1 = map.AddAction("action1");
+        /// var action2 = map.AddAction("action2");
+        ///
+        /// // Add a binding to each action to which  a "ClampProcessor" is applied.
+        /// // This processor has two parameters:
+        /// // - "min" (float)
+        /// // - "max" (float)
+        /// action1.AddBinding("&gt;Gamepad&gt;/rightTrigger", processors: "clamp(min=0.2,max=0.8)");
+        /// action2.AddBinding("&gt;Gamepad&gt;/leftTrigger", processors: "clamp(min=0.2,max=0.8)");
+        ///
+        /// // Apply parameter overrides to set the values differently.
+        /// // This will apply the setting to *both* the bindings on action1 *and* action2.
+        /// map.ApplyParameterOverride("min", 0.3f);
+        /// map.ApplyParameterOverride("max", 0.9f);
+        /// </code>
+        /// </example>
+        ///
+        /// An override can optionally be directed at a specific type of object.
+        ///
+        /// <example>
+        /// <code>
+        /// map.ApplyParameterOverride("clamp:min", 0.3f);
+        /// map.ApplyParameterOverride("clamp:max", 0.9f);
+        /// </code>
+        /// </example>
+        ///
+        /// By default, the parameter override will apply to all bindings in the map. To limit the override
+        /// to specific bindings, you can supply a <paramref name="bindingMask"/>.
+        ///
+        /// <example>
+        /// <code>
+        /// // Apply a parameter override only to action1.
+        /// map.ApplyBindingOverride("clamp:min", 0.25f, new InputBinding { action = action1.name });
+        ///
+        /// // Apply a parameter override only to a specific binding path.
+        /// map.ApplyBindingOverride("clamp:min", 0.4f, new InputBinding { path = "&lt;Gamepad&gt;/leftTrigger" });
+        /// </code>
+        /// </example>
+        ///
+        /// If there are multiple overrides that could be applied to a specific parameter, the most specific override
```

Comment:
I find this paragraph a bit confusing, maybe possible to rephrase this a bit to make it easier to grasp? We are talking about two overrides "duration", but what do we mean by "an override for "duration" specifically on bindings to the "look" action exists"?

Reply:
Took a stab at rewording this. Hopefully clearer now.